benchmark/benchmark_config.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __BENCHMARK_BENCHMARK_CONFIG_H__
+#define __BENCHMARK_BENCHMARK_CONFIG_H__
+
+#define USE_HTTPS
+
+#ifdef USE_HTTPS
```

Comment:
`USE_HTTPS` is not needed, it can simply use `IS_HTTPS` with a description. 
```
#defined IS_HTTPS 0 // 0 for HTTP, 1 for HTTPS
```
Node and Faucet configs are duplicated.

Reply:
HTTPS was completely removed from a Benchmark module configuration.